CHANGES.md

```diff
@@ -1,3 +1,33 @@
+1.7.2 / 2019-09-XX
+==================
+
+Bug fixes:
+
+* DriverInterface#getValue() can return null
+* Modify response[Contains|NotContains] to use strpos vs preg_match
+* Fix compatibility with PHP 7.2
+* Allow Symfony 4.
+* Optimize the splitting of XPath unions
```

Comment:
Please remove this one.

---

Historically there were 2 PRs:

* in 1st one support for unions was added
* in 2nd one that support was improved

For end user these looks as a single change since it happened inside same version.